The app keeps crashing since the new update 0.16.3

### Describe the bug

The app opens but can`t play anything, it crashes. 

### Logs

---
client: Jellyfin for Android TV
client_version: 0.16.3
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out
	at org.jellyfin.sdk.api.client.KtorClient.request$suspendImpl(KtorClient.kt:117)
	at org.jellyfin.sdk.api.client.KtorClient$request$1.invokeSuspend(Unknown Source:19)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7839)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@8e48970, Dispatchers.Main.immediate]
Caused by: io.ktor.client.features.HttpRequestTimeoutException: Request timeout has expired [url=https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Items/Resume?limit=50&fields=PrimaryImageAspectRatio&fields=Overview&fields=ItemCounts&fields=DisplayPreferencesId&fields=ChildCount&mediaTypes=Video&imageTypeLimit=1&enableTotalRecordCount=false&enableImages=true&excludeActiveSessions=false, request_timeout=30000 ms]
	at io.ktor.client.features.HttpTimeout$Feature$install$1$1$killer$1.invokeSuspend(HttpTimeout.kt:149)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)

```  
***Logcat***: 
```log
01-06 19:32:16.491 D/Compose Focus(20815): Owner FocusChanged(false)
01-06 19:32:16.496 I/OpenGLRenderer(20815): Davey! duration=30265ms; Flags=0, FrameTimelineVsyncId=23639541, IntendedVsync=6554537519970451, Vsync=6554567683436515, InputEventId=0, HandleInputStart=6554567697175079, AnimationStart=6554567697177371, PerformTraversalsStart=6554567759828704, DrawStart=6554567760298037, FrameDeadline=6554537653468324, FrameInterval=6554567697013329, FrameStartTime=16683333, SyncQueued=6554567761702162, SyncStart=6554567761822912, IssueDrawCommandsStart=6554567762043662, SwapBuffers=6554567780444371, FrameCompleted=6554567785451454, DequeueBufferDuration=24209, QueueBufferDuration=957083, GpuCompleted=6554567785451454, SwapBuffersCompleted=6554567782201954, DisplayPresentTime=1394826913584475024, 
01-06 19:32:16.525 I/jellyfin-apiclient(20815): Response received from: https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Views?format=json
01-06 19:32:16.587 D/ACRA    (20815): Collector org.acra.collector.LogCatCollector completed
01-06 19:32:16.587 D/ACRA    (20815): Finished collectors with priority FIRST
01-06 19:32:16.587 D/ACRA    (20815): Starting collectors with priority NORMAL
01-06 19:32:16.587 D/ACRA    (20815): Calling collector org.acra.collector.ConfigurationCollector
01-06 19:32:16.589 D/ACRA    (20815): Calling collector org.acra.collector.DeviceFeaturesCollector
01-06 19:32:16.589 D/ACRA    (20815): Calling collector org.acra.collector.CustomDataCollector
01-06 19:32:16.589 D/ACRA    (20815): Collector org.acra.collector.CustomDataCollector completed
01-06 19:32:16.590 D/ACRA    (20815): Calling collector org.acra.collector.PackageManagerCollector
01-06 19:32:16.590 D/ACRA    (20815): Calling collector org.acra.collector.SettingsCollector
01-06 19:32:16.591 D/ACRA    (20815): Collector org.acra.collector.SettingsCollector completed
01-06 19:32:16.591 D/ACRA    (20815): Collector org.acra.collector.PackageManagerCollector completed
01-06 19:32:16.595 D/ACRA    (20815): Calling collector org.acra.collector.DisplayManagerCollector
01-06 19:32:16.595 D/ACRA    (20815): Calling collector org.acra.collector.DeviceIdCollector
01-06 19:32:16.595 D/ACRA    (20815): Calling collector org.acra.collector.MemoryInfoCollector
01-06 19:32:16.595 D/ACRA    (20815): Collector org.acra.collector.DeviceIdCollector completed
01-06 19:32:16.596 D/ACRA    (20815): Calling collector org.acra.collector.ReflectionCollector
01-06 19:32:16.596 D/ACRA    (20815): Calling collector org.acra.collector.TimeCollector
01-06 19:32:16.597 D/ACRA    (20815): Collector org.acra.collector.TimeCollector completed
01-06 19:32:16.597 D/ACRA    (20815): Collector org.acra.collector.DeviceFeaturesCollector completed
01-06 19:32:16.600 D/ACRA    (20815): Collector org.acra.collector.DisplayManagerCollector completed
01-06 19:32:16.602 D/ACRA    (20815): Calling collector org.acra.collector.SimpleValuesCollector
01-06 19:32:16.604 D/ACRA    (20815): Calling collector org.acra.collector.SharedPreferencesCollector
01-06 19:32:16.604 D/ACRA    (20815): Collector org.acra.collector.SharedPreferencesCollector completed
01-06 19:32:16.604 D/ACRA    (20815): Collector org.acra.collector.SimpleValuesCollector completed
01-06 19:32:16.608 D/ACRA    (20815): Collector org.acra.collector.ConfigurationCollector completed
01-06 19:32:16.617 D/ItemRowAdapterHelperKt(20815): Creating items from 0 existing and 2 new, adapter size is 0
01-06 19:32:16.625 D/CompatibilityChangeReporter(20815): Compat change id reported: 149924527; UID 10090; state: ENABLED
01-06 19:32:16.626 D/CompatibilityChangeReporter(20815): Compat change id reported: 132649864; UID 10090; state: DISABLED
01-06 19:32:16.630 D/ACRA    (20815): Collector org.acra.collector.MemoryInfoCollector completed
01-06 19:32:16.636 D/ACRA    (20815): Collector org.acra.collector.ReflectionCollector completed
01-06 19:32:16.636 D/ACRA    (20815): Finished collectors with priority NORMAL
01-06 19:32:16.637 D/ACRA    (20815): Starting collectors with priority LATE
01-06 19:32:16.637 D/ACRA    (20815): Calling collector org.acra.collector.MediaCodecListCollector
01-06 19:32:16.637 D/ACRA    (20815): Calling collector org.acra.collector.ThreadCollector
01-06 19:32:16.637 D/ACRA    (20815): Calling collector org.acra.collector.LogFileCollector
01-06 19:32:16.637 D/ACRA    (20815): Collector org.acra.collector.MediaCodecListCollector completed
01-06 19:32:16.637 D/ACRA    (20815): Collector org.acra.collector.LogFileCollector completed
01-06 19:32:16.637 D/ACRA    (20815): Collector org.acra.collector.ThreadCollector completed
01-06 19:32:16.637 D/ACRA    (20815): Finished collectors with priority LATE
01-06 19:32:16.637 D/ACRA    (20815): Finishing activities prior to killing the Process
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): HTTP request timed out
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): io.ktor.client.features.HttpRequestTimeoutException: Request timeout has expired [url=https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Items/Resume?limit=50&fields=PrimaryImageAspectRatio&fields=Overview&fields=ItemCounts&fields=DisplayPreferencesId&fields=ChildCount&mediaTypes=Video&imageTypeLimit=1&enableTotalRecordCount=false&enableImages=true&excludeActiveSessions=false, request_timeout=30000 ms]
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at io.ktor.client.features.HttpTimeout$Feature$install$1$1$killer$1.invokeSuspend(HttpTimeout.kt:149)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
01-06 19:32:16.638 D/org.jellyfin.sdk.api.client.KtorClient(20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
01-06 19:32:16.639 E/ACRA    (20815): ACRA caught a TimeoutException for org.jellyfin.androidtv
01-06 19:32:16.639 E/ACRA    (20815): org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out
01-06 19:32:16.639 E/ACRA    (20815): 	at org.jellyfin.sdk.api.client.KtorClient.request$suspendImpl(KtorClient.kt:117)
01-06 19:32:16.639 E/ACRA    (20815): 	at org.jellyfin.sdk.api.client.KtorClient$request$1.invokeSuspend(Unknown Source:19)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
01-06 19:32:16.639 E/ACRA    (20815): 	at android.os.Handler.handleCallback(Handler.java:938)
01-06 19:32:16.639 E/ACRA    (20815): 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-06 19:32:16.639 E/ACRA    (20815): 	at android.os.Looper.loopOnce(Looper.java:201)
01-06 19:32:16.639 E/ACRA    (20815): 	at android.os.Looper.loop(Looper.java:288)
01-06 19:32:16.639 E/ACRA    (20815): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
01-06 19:32:16.639 E/ACRA    (20815): 	at java.lang.reflect.Method.invoke(Native Method)
01-06 19:32:16.639 E/ACRA    (20815): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
01-06 19:32:16.639 E/ACRA    (20815): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
01-06 19:32:16.639 E/ACRA    (20815): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@8e48970, Dispatchers.Main.immediate]
01-06 19:32:16.639 E/ACRA    (20815): Caused by: io.ktor.client.features.HttpRequestTimeoutException: Request timeout has expired [url=https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Items/Resume?limit=50&fields=PrimaryImageAspectRatio&fields=Overview&fields=ItemCounts&fields=DisplayPreferencesId&fields=ChildCount&mediaTypes=Video&imageTypeLimit=1&enableTotalRecordCount=false&enableImages=true&excludeActiveSessions=false, request_timeout=30000 ms]
01-06 19:32:16.639 E/ACRA    (20815): 	at io.ktor.client.features.HttpTimeout$Feature$install$1$1$killer$1.invokeSuspend(HttpTimeout.kt:149)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
01-06 19:32:16.639 E/ACRA    (20815): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
01-06 19:32:16.639 D/ACRA    (20815): Building report
01-06 19:32:16.640 D/ACRA    (20815): Starting collectors with priority FIRST
01-06 19:32:16.641 D/ACRA    (20815): Calling collector org.acra.collector.DropBoxCollector
01-06 19:32:16.641 D/ACRA    (20815): Collector org.acra.collector.DropBoxCollector completed
01-06 19:32:16.641 D/ACRA    (20815): Calling collector org.acra.collector.LogCatCollector
01-06 19:32:16.642 D/ACRA    (20815): Calling collector org.acra.collector.StacktraceCollector
01-06 19:32:16.644 D/ACRA    (20815): Collector org.acra.collector.StacktraceCollector completed
01-06 19:32:16.653 D/ACRA    (20815): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.16.3` (`160399`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"boreal","BOOTLOADER":"01.01.230615.104351","BRAND":"google","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"boreal","DISPLAY":"STTK.230808.004","FINGERPRINT":"google\/boreal\/boreal:12\/STTK.230808.004\/10714782:user\/release-keys","HARDWARE":"amlogic","HOST":"abfarm-2004-4097","ID":"STTK.230808.004","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Google","MODEL":"Chromecast HD","ODM_SKU":"unknown","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"boreal","RADIO":"unknown","SKU":"CA","SOC_MANUFACTURER":"Amlogic","SOC_MODEL":"AMLS805X2","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1692975634000,"TYPE":"user","UNKNOWN":"unknown","USER":"android-build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"10714782","MEDIA_PERFORMANCE_CLASS":0,"PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"12","RELEASE_OR_CODENAME":"12","SDK":"31","SDK_INT":31,"SECURITY_PATCH":"2023-09-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160399,"VERSION_NAME":"0.16.3"}
```  

### Device information

***Android version***: `12`  
***Device brand***: `google`  
***Device product***: `boreal`  
***Device model***: `Chromecast HD`  

### Crash information

***Start time***: `2024-01-06T19:31:38.092-05:00`  
***Crash time***: `2024-01-06T19:32:16.852-05:00`  


### Application version

0.16.3

### Where did you install the app from?

Google Play

### Device information

Chromecast HD

### Android version

12

### Jellyfin server version

10.8.13

``---
client: Jellyfin for Android TV
client_version: 0.16.3
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException: Attempt to invoke virtual method 'void org.jellyfin.androidtv.ui.presentation.MyDetailsOverviewRowPresenter$ViewHolder.setInfoValue3(java.lang.String)' on a null object reference
	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$4.run(FullDetailsFragment.java:339)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7839)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

```  
***Logcat***: 
```log
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           ... 14 more
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       Caused by: java.lang.UnsupportedOperationException: Cannot create AudioTrack
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           at android.media.AudioTrack$Builder.build(AudioTrack.java:1308)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           at androidx.media3.exoplayer.drm.FrameworkMediaDrm$$ExternalSyntheticApiModelOutline0.m(Unknown Source:0)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.createAudioTrackV29(DefaultAudioSink.java:2134)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.createAudioTrack(DefaultAudioSink.java:2113)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.buildAudioTrack(DefaultAudioSink.java:2079)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):           ... 13 more
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):   Caused by: java.lang.UnsupportedOperationException: Cannot create AudioTrack
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       at android.media.AudioTrack$Builder.build(AudioTrack.java:1308)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       at androidx.media3.exoplayer.drm.FrameworkMediaDrm$$ExternalSyntheticApiModelOutline0.m(Unknown Source:0)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.createAudioTrackV29(DefaultAudioSink.java:2134)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.createAudioTrack(DefaultAudioSink.java:2113)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       at androidx.media3.exoplayer.audio.DefaultAudioSink$Configuration.buildAudioTrack(DefaultAudioSink.java:2079)
01-06 19:29:30.104 E/ExoPlayerImplInternal(20092):       ... 13 more
01-06 19:29:30.125 D/MediaCodec(20092): keep callback message for reclaim
01-06 19:29:30.132 D/CCodecBuffers(20092): [c2.android.aac.decoder#751:1D-Output.Impl[N]] Client returned a buffer it does not own according to our record: 0
01-06 19:29:30.132 D/CCodecBuffers(20092): [c2.android.aac.decoder#751:1D-Output.Impl[N]] Client returned a buffer it does not own according to our record: 1
01-06 19:29:30.132 D/CCodecBuffers(20092): [c2.android.aac.decoder#751:1D-Output.Impl[N]] Client returned a buffer it does not own according to our record: 2
01-06 19:29:30.132 D/CCodecBuffers(20092): [c2.android.aac.decoder#751:1D-Output.Impl[N]] Client returned a buffer it does not own according to our record: 3
01-06 19:29:30.133 D/MediaCodec(20092): keep callback message for reclaim
01-06 19:29:30.133 I/CCodecConfig(20092): query failed after returning 19 values (BAD_INDEX)
01-06 19:29:30.134 W/Codec2Client(20092): query -- param skipped: index = 1342179345.
01-06 19:29:30.134 W/Codec2Client(20092): query -- param skipped: index = 2415921170.
01-06 19:29:30.134 W/Codec2Client(20092): query -- param skipped: index = 1610614798.
01-06 19:29:30.137 D/SurfaceUtils(20092): connecting to surface 0xe6f72618, reason connectToSurface
01-06 19:29:30.137 I/MediaCodec(20092): [OMX.amlogic.avc.decoder.awesome2] setting surface generation to 20574226
01-06 19:29:30.137 D/SurfaceUtils(20092): disconnecting from surface 0xe6f72618, reason connectToSurface(reconnect)
01-06 19:29:30.137 D/SurfaceUtils(20092): connecting to surface 0xe6f72618, reason connectToSurface(reconnect)
01-06 19:29:30.138 W/ACodec  (20092): cannot change tunneled surface
01-06 19:29:30.138 W/MediaCodec(20092): error setting release surface: err = -38
01-06 19:29:30.140 D/CCodecBufferChannel(20092): [c2.android.aac.decoder#751] MediaCodec discarded an unknown buffer
01-06 19:29:30.140 D/CCodecBufferChannel(20092): [c2.android.aac.decoder#751] MediaCodec discarded an unknown buffer
01-06 19:29:30.140 D/CCodecBufferChannel(20092): [c2.android.aac.decoder#751] MediaCodec discarded an unknown buffer
01-06 19:29:30.140 D/CCodecBufferChannel(20092): [c2.android.aac.decoder#751] MediaCodec discarded an unknown buffer
01-06 19:29:30.140 I/hw-BpHwBinder(20092): onLastStrongRef automatically unlinking death recipients
01-06 19:29:30.142 D/MediaCodec(20092): flushMediametrics
01-06 19:29:30.143 D/MediaCodec(20092): flushMediametrics
01-06 19:29:30.146 D/VideoManager(20092): Caught player timeline change - reason: SOURCE_UPDATE
01-06 19:29:30.148 E/VideoManager(20092): ***** Got error from player
01-06 19:29:30.149 E/PlaybackController(20092): Playback error - Failed to load video
01-06 19:29:30.153 D/NavigationRepositoryImpl(20092): Navigating back
01-06 19:29:30.176 D/CustomPlaybackOverlayFragment(20092): Stopping!
01-06 19:29:30.177 D/CustomPlaybackOverlayFragment(20092): this fragment belongs to the current session, ending it
01-06 19:29:30.178 D/PlaybackController(20092): stop called at 0
01-06 19:29:30.178 I/ReportingHelper(20092): ReportingHelper.reportStopped called for cbc91bcb-dd56-21c9-a378-8c50cf7b7245 at position 0
01-06 19:29:30.179 I/jellyfin-apiclient(20092): ReportPlaybackStopped: Item cbc91bcb-dd56-21c9-a378-8c50cf7b7245, Ticks: 0
01-06 19:29:30.180 D/jellyfin-apiclient(20092): Adding request to queue: https://connect.streamii.box.ca/Sessions/Playing/Stopped
01-06 19:29:30.191 I/ExoPlayerImpl(20092): Release be141b7 [AndroidXMedia3/1.2.0] [boreal, Chromecast HD, Google, 31] [media3.common, media3.exoplayer, media3.decoder, media3.session, media3.ui, media3.datasource, media3.extractor, media3.decoder.ffmpeg]
01-06 19:29:30.220 D/SurfaceUtils(20092): disconnecting from surface 0xe6f78ad8, reason disconnectFromSurface
01-06 19:29:30.222 D/MediaCodec(20092): flushMediametrics
01-06 19:29:30.224 D/MediaCodec(20092): flushMediametrics
01-06 19:29:30.230 D/jellyfin-apiclient(20092): Adding request to queue: https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Items/112edf60-c1f7-b096-267c-a650f9e06458?format=json
01-06 19:29:30.275 W/RecyclerView(20092): No adapter attached; skipping layout
01-06 19:29:30.365 D/BufferPoolAccessor2.0(20092): bufferpool2 0xbd4d6828 : 0(0 size) total buffers - 0(0 size) used buffers - 14/19 (recycle/alloc) - 5/14 (fetch/transfer)
01-06 19:29:30.365 D/BufferPoolAccessor2.0(20092): evictor expired: 1, evicted: 1
01-06 19:29:31.019 D/FullDetailsFragment(20092): current time 1704587371018 last playback event time 1704587370181 last refresh time 1704587370233
01-06 19:29:31.031 D/FullDetailsFragment(20092): Updating info after playback
01-06 19:29:31.032 D/jellyfin-apiclient(20092): Adding request to queue: https://connect.streamii.box.ca/Users/a3252c9f-cfa9-47ce-8313-1c344efb4d54/Items/112edf60c1f7b096267ca650f9e06458?format=json
01-06 19:29:31.365 D/BufferPoolAccessor2.0(20092): evictor expired: 1, evicted: 0
01-06 19:29:32.365 D/BufferPoolAccessor2.0(20092): bufferpool2 0xbd4d8b08 : 0(0 size) total buffers - 0(0 size) used buffers - 13/18 (recycle/alloc) - 5/13 (fetch/transfer)
01-06 19:29:32.366 D/BufferPoolAccessor2.0(20092): evictor expired: 1, evicted: 1
01-06 19:29:35.366 D/BufferPoolAccessor2.0(20092): bufferpool2 0xbd4c2548 : 0(0 size) total buffers - 0(0 size) used buffers - 14/19 (recycle/alloc) - 5/14 (fetch/transfer)
01-06 19:29:35.366 D/BufferPoolAccessor2.0(20092): evictor expired: 1, evicted: 1
01-06 19:29:39.671 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(20092): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
01-06 19:29:39.711 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(20092): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"a79c7bbfc7014d129598e960296756a7"}
01-06 19:29:45.287 D/AndroidRuntime(20092): Shutting down VM
--------- beginning of crash
01-06 19:29:45.299 E/AndroidRuntime(20092): FATAL EXCEPTION: main
01-06 19:29:45.299 E/AndroidRuntime(20092): Process: org.jellyfin.androidtv, PID: 20092
01-06 19:29:45.299 E/AndroidRuntime(20092): java.lang.NullPointerException: Attempt to invoke virtual method 'void org.jellyfin.androidtv.ui.presentation.MyDetailsOverviewRowPresenter$ViewHolder.setInfoValue3(java.lang.String)' on a null object reference
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$4.run(FullDetailsFragment.java:339)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at android.os.Handler.handleCallback(Handler.java:938)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at android.os.Looper.loopOnce(Looper.java:201)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at android.os.Looper.loop(Looper.java:288)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at java.lang.reflect.Method.invoke(Native Method)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
01-06 19:29:45.299 E/AndroidRuntime(20092): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
01-06 19:29:45.301 E/ACRA    (20092): ACRA caught a NullPointerException for org.jellyfin.androidtv
01-06 19:29:45.301 E/ACRA    (20092): java.lang.NullPointerException: Attempt to invoke virtual method 'void org.jellyfin.androidtv.ui.presentation.MyDetailsOverviewRowPresenter$ViewHolder.setInfoValue3(java.lang.String)' on a null object reference
01-06 19:29:45.301 E/ACRA    (20092): 	at org.jellyfin.androidtv.ui.itemdetail.FullDetailsFragment$4.run(FullDetailsFragment.java:339)
01-06 19:29:45.301 E/ACRA    (20092): 	at android.os.Handler.handleCallback(Handler.java:938)
01-06 19:29:45.301 E/ACRA    (20092): 	at android.os.Handler.dispatchMessage(Handler.java:99)
01-06 19:29:45.301 E/ACRA    (20092): 	at android.os.Looper.loopOnce(Looper.java:201)
01-06 19:29:45.301 E/ACRA    (20092): 	at android.os.Looper.loop(Looper.java:288)
01-06 19:29:45.301 E/ACRA    (20092): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
01-06 19:29:45.301 E/ACRA    (20092): 	at java.lang.reflect.Method.invoke(Native Method)
01-06 19:29:45.301 E/ACRA    (20092): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
01-06 19:29:45.301 E/ACRA    (20092): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
01-06 19:29:45.301 D/ACRA    (20092): Building report
01-06 19:29:45.306 D/ACRA    (20092): Starting collectors with priority FIRST
01-06 19:29:45.307 D/ACRA    (20092): Calling collector org.acra.collector.DropBoxCollector
01-06 19:29:45.307 D/ACRA    (20092): Calling collector org.acra.collector.StacktraceCollector
01-06 19:29:45.307 D/ACRA    (20092): Calling collector org.acra.collector.LogCatCollector
01-06 19:29:45.307 D/ACRA    (20092): Collector org.acra.collector.DropBoxCollector completed
01-06 19:29:45.311 D/ACRA    (20092): Collector org.acra.collector.StacktraceCollector completed
01-06 19:29:45.327 D/ACRA    (20092): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.16.3` (`160399`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"boreal","BOOTLOADER":"01.01.230615.104351","BRAND":"google","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"boreal","DISPLAY":"STTK.230808.004","FINGERPRINT":"google\/boreal\/boreal:12\/STTK.230808.004\/10714782:user\/release-keys","HARDWARE":"amlogic","HOST":"abfarm-2004-4097","ID":"STTK.230808.004","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Google","MODEL":"Chromecast HD","ODM_SKU":"unknown","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"boreal","RADIO":"unknown","SKU":"CA","SOC_MANUFACTURER":"Amlogic","SOC_MODEL":"AMLS805X2","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1692975634000,"TYPE":"user","UNKNOWN":"unknown","USER":"android-build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"10714782","MEDIA_PERFORMANCE_CLASS":0,"PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"12","RELEASE_OR_CODENAME":"12","SDK":"31","SDK_INT":31,"SECURITY_PATCH":"2023-09-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160399,"VERSION_NAME":"0.16.3"}
```  

### Device information

***Android version***: `12`  
***Device brand***: `google`  
***Device product***: `boreal`  
***Device model***: `Chromecast HD`  

### Crash information

***Start time***: `2024-01-06T19:22:36.423-05:00`  
***Crash time***: `2024-01-06T19:29:45.552-05:00`

--- a/app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java
@@ -208,7 +208,6 @@ public class VideoManager implements IVLCVout.OnNewVideoLayoutListener {
 
         DefaultTrackSelector trackSelector = new DefaultTrackSelector(context);
         trackSelector.setParameters(trackSelector.buildUponParameters()
-                .setTunnelingEnabled(true)
                 .setAudioOffloadPreferences(new TrackSelectionParameters.AudioOffloadPreferences.Builder()
                         .setAudioOffloadMode(TrackSelectionParameters.AudioOffloadPreferences.AUDIO_OFFLOAD_MODE_ENABLED)
                         .build()

--- a/playback/exoplayer/src/main/kotlin/ExoPlayerBackend.kt
+++ b/playback/exoplayer/src/main/kotlin/ExoPlayerBackend.kt
@@ -49,7 +49,6 @@ class ExoPlayerBackend(
 			})
 			.setTrackSelector(DefaultTrackSelector(context).apply {
 				setParameters(buildUponParameters().apply {
-					setTunnelingEnabled(true)
 					setAudioOffloadPreferences(TrackSelectionParameters.AudioOffloadPreferences.DEFAULT.buildUpon().apply {
 						setAudioOffloadMode(TrackSelectionParameters.AudioOffloadPreferences.AUDIO_OFFLOAD_MODE_ENABLED)
 					}.build())

